web/packages/core/src/public/player/flash.ts

```diff
@@ -3,15 +3,58 @@
  * These don't seem to be documented in full anywhere, and Ruffle adds them as we encounter some.
  * You are discouraged from using these, and they exist only to support legacy websites from decades ago.
  *
+ * https://web.archive.org/web/20100710000820/http://www.adobe.com/support/flash/publishexport/scriptingwithflash/scriptingwithflash_03.html
+ *
  * Extra methods or properties may appear at any time, due to `ExternalInterface.addCallback()`.
  * It may even overwrite existing methods or properties.
  */
 export interface FlashAPI {
+    /**
+     * Returns the current frame index of the movie.
+     *
+     * @remarks the frame number starts from a 0 index, not 1 like in FP
```

Comment:
To be clear, if this a Ruffle vs Flash thing or a Flash JavaScript vs ActionScript thing?

Is this method supposed to be 0 indexed in the Flash JS API?

(Same for other frame based methods)

Reply:
It might be clearer to change this (and other instances of this comment) to:

> the frame number starts from a 0 index, not 1 like in AS